README.md

```diff
@@ -32,7 +32,16 @@ By default, the `latest` version of Atlas is being used. If you want to lock a s
         with:
             version: "vX.Y.Z"
 ```
+### Cloud Token
 
+Some Atlas features require cloud integration, the `cloud-token` is used to authenticate with Atlas Cloud. More information about atlas cloud can be found [here](https://atlasgo.io/cloud/getting-started)
```

Comment:
Use the actual param as used [here](https://github.com/ariga/atlas-sync-action#configuration)
```suggestion
### cloud-token

API token to authenticate to an Atlas Cloud organization. [Read more](https://atlasgo.io/cloud/getting-started)
```